app/views/tracking/UnknownPage.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import {
+  Dimensions,
+  ImageBackground,
+  StatusBar,
+  Text,
+  View,
+} from 'react-native';
+import { openSettings } from 'react-native-permissions';
+import { SvgXml } from 'react-native-svg';
+
+import BackgroundImage from './../../assets/images/launchScreenBackground.png';
+import StateUnknown from './../../assets/svgs/stateUnknown';
+import { Typography } from './../../components/Typography';
+import Colors from './../../constants/colors';
+import ButtonWrapper from '../../components/ButtonWrapper';
+import languages from '../../locales/languages';
+import MayoInfo from './MayoInfo';
+import styles from './style';
+
+const buttonLabel = languages.t('label.home_enable_location');
```

Comment:
use `const {t} = useTranslation()` inside the functional component. because its reactive.

Reply:
the default export of the languages file uses the `useTranslation()` hook